lib/common/delete-phases-common.js

```diff
@@ -59,9 +60,16 @@ exports.unDeployCloudFormationStack = function (serviceContext, serviceType) {
             }
         })
         .then(() => {
+          return s3Calls.deleteMatchingPrefix('handel-'+accountConfig.region+'-'+accountConfig.account_id,serviceContext.appName+'/'+serviceContext.environmentName+'/'+serviceContext.serviceName);
```

Comment:
Could we split this out into two variables that get referenced in this function? Also can we use the ES6 string interpolation syntax? That will help it be a bit more understandable for people reading it after the fact I think. Here's what I'm thinking of:

```
let bucketName = `handel-${accountConfig.region}-${accountConfig.account_id}`;
let prefix = `${serviceContext.appName}/${serviceContext.environmentName}/${serviceContext.serviceName}`;
return s3Calls.deleteMatchingPrefix(bucketName, prefix);      
```